misc/semver-checker/modules/semver-checker-core/src/main/java/io/ballerina/semver/checker/SemverChecker.java

```diff
@@ -109,12 +100,12 @@ public Optional<PackageDiff> computeDiff() throws SemverToolException {
         String pkgName = currentPackage.packageName().value();
         SemanticVersion pkgVersion = currentPackage.packageVersion().value();
 
-        CentralClientWrapper clientWrapper = new CentralClientWrapper(outStream, errStream);
-        if (releasedVersion == null) {
-            errStream.println("checking for the latest compatible release version available in central...");
-            releasedVersion = clientWrapper.getLatestCompatibleVersion(orgName, pkgName, pkgVersion);
+        BallerinaPackageResolver pkgResolver = new BallerinaPackageResolver(outStream, errStream);
```

Comment:
I'm curious whether we use `errStream` to print errors?

Reply:
Yes we use it to output the errors.